src/web/routes/application/are-you-pregnant/validate.js

```diff
@@ -50,7 +50,7 @@ const callValidateExpectedDeliveryDate = (req, res, next) =>
   check('expectedDeliveryDate').custom(validateExpectedDeliveryDate(res))(req, res, next)
 
 const validate = [
-  validateIsYesOrNo('areYouPregnant'),
+  validateIsYesOrNo('areYouPregnant', 'validation:selectYesOrNo'),
```

Comment:
I'll leave this one until we know what the Are You Pregnant one should be, I don't think there's a story for it yet.